src/utils/filings.ts

```diff
@@ -0,0 +1,362 @@
+//
+// Filing Type helpers
+//
+import { FilingTypes } from '@bcrs-shared-components/enums'
+import type { ApiResponseFilingI } from '~/interfaces/filing-i'
+import type { StateFilingI } from '~/interfaces/business-i'
+
+export class FilingUtils {
+  /** Returns True if filing is an AGM Extension. */
+  static isTypeAgmExtension (filing: ApiResponseFilingI): boolean {
```

Comment:
I discussed this with patrick in his PR as well -- there will be a couple hundred filing types eventually and having an is... function for every one is going to be unweildy and hard to navigate. Can we have one function that takes the filing / filing type and returns true / false?